src/render/filters.rs

```diff
@@ -541,7 +541,7 @@ impl ResolveFilter for YesnoFilter {
             None => no,
         };
 
-        Ok(Some(result.to_string().into_content()))
+        Ok(Some((*result).to_string().into_content()))
```

Comment:
I'm not sure we need to fix this one - it [only affects old compiler versions](https://rust-lang.github.io/rust-clippy/master/index.html?search=inefficient_to_string) which I don't think we care to support.

Reply:
oh, the documentation changed on master. This was not mentioned for rust [1.91](https://rust-lang.github.io/rust-clippy/rust-1.91.0/index.html?search=inefficient_to_string#/inefficient_to_string)

I'm ignoring this one then